src/main.js

```diff
@@ -189,7 +187,7 @@ function takeBackToMain(page) {
 };
 
 function displayMyPoster() {
-  event.preventDefault();
+  event.preventDefault(event);
   mainPosterSection.classList.remove('hidden');
```

Comment:
Consider adding a function to main.js that accepts an HTML element then applies the 'hidden' class. Something like...
```
function hide(element) {
//apply hidden class to element parameter
}
```
Then you can reuse that function anytime you want to hide something. You could do the same thing to for revealing stuff (function reveal(element) which removes the hidden class). 

element.classList.toggle is also helpful. It will add a class if an element doesn't have it or remove it if it does.

Reply:
Super helpful feedback, Matt! We went ahead and made those changes and everything is still working great. Appreciate it!